screenpy/__init__.py

```diff
@@ -19,28 +19,170 @@
 :license: MIT, see LICENSE for more details.
 """
 
+from .actions import (
+    Assert,
+    AssertAllOf,
+    AssertAnyOf,
+    AttachAFile,
+    AttachFile,
+    AttachTheFile,
+    Confirm,
+    ConfirmAllOf,
+    ConfirmAnyOf,
+    Debug,
+    Eventually,
+    MakeNote,
+    Observe,
+    ObserveAllOf,
+    ObserveAnyOf,
+    Pause,
+    See,
+    SeeAllOf,
+    SeeAnyOf,
+    Sleep,
+    TakeNote,
+    Verify,
+    VerifyAllOf,
+    VerifyAnyOf,
+)
 from .actor import Actor
 from .directions import noted, noted_under, the_noted
 from .director import Director
 from .given_when_then import and_, given, given_that, then, when
+from .narration import Narrator
 from .pacing import act, aside, beat, scene, the_narrator
+from .resolutions import (
+    ContainItemMatching,
+    ContainsItemMatching,
+    ContainsTheEntries,
+    ContainsTheEntry,
+    ContainsTheItem,
+    ContainsTheKey,
+    ContainsTheText,
+    ContainsTheValue,
+    ContainTheEntries,
+    ContainTheEntry,
+    ContainTheItem,
+    ContainTheKey,
+    ContainTheText,
+    ContainTheValue,
+    DoesNot,
+    DoNot,
+    Empty,
+    EndsWith,
+    EndWith,
+    Equal,
+    Equals,
+    EqualTo,
+    GreaterThan,
+    GreaterThanOrEqualTo,
+    HasLength,
+    InRange,
+    IsCloseTo,
+    IsEmpty,
+    IsEqual,
+    IsEqualTo,
+    IsGreaterThan,
+    IsGreaterThanOrEqualTo,
+    IsInRange,
+    IsLessThan,
+    IsLessThanOrEqualTo,
+    IsNot,
+    LessThan,
+    LessThanOrEqualTo,
+    Match,
+    Matches,
+    ReadExactly,
+    ReadsExactly,
+    StartsWith,
+    StartWith,
+)
 
 # Natural-language-enabling syntactic sugar
 AnActor = Actor
 
 
 __all__ = [
```

Comment:
This is alphabetical using the same rules as `sorted(expected)`.

Reply:
This is sorted by isort.

Reply:
What, really? The `__all__` list gets sorted by `isort`, too?

Reply:
Sorry.  I thought I updated that comment.  The sorted strings in `__all__` are from python's `sorted()` functionality.  isort only sorts the imports.

Reply:
I think i'd prefer just alphabetized rather than `sorted`, that will make maintenance easier. I can just press `F5` to sort a selection of lines, whereas with `sorted` i'd need to copy the whole list, paste it into Python, sort it, copy the result, paste it back, and probably edit the formatting. :P

Reply:
That's funny, because I do something similar and it sorts it this way in pycharm.

Reply:
Haha. I guess it'll just be sorted whichever way by whoever more recently added the next Action. :P

I won't make you change it if that's how it works for your IDE, i thought you had taken pains to make sure it was sorted the same way as `sorted` would do it.